### PR TITLE
boot-utils: Add support for RISCV

### PR DESCRIPTION
**Pull request by nathanchance**

### CODE REVIEW

File: boot-qemu.sh [hunk 1]
```diff
@@ -186,6 +186,15 @@ function setup_qemu_args() {
             QEMU=(qemu-system-ppc64)
             ;;
 
+        riscv)
+            KIMAGE=Image
+            QEMU_ARCH_ARGS=(
+                -bios default
```
Comment (nickdesaulniers) [inline boot-qemu.sh:192]: does this work? I seem to have `-bios /usr/lib/riscv64-linux-gnu/opensbi/qemu/virt/fw_jump.elf` in my shell history.

Comment (nickdesaulniers) [reply boot-qemu.sh:192]: If I use `-bios default` locally, I see:
```
qemu-system-riscv64: Unable to load the RISC-V firmware "opensbi-riscv64-virt-fw_jump.bin"
```

Comment (nathanchance) [reply boot-qemu.sh:192]: It works for me but I build and use QEMU from source. I'll see if I can test with just stock Ubuntu or Debian packages.

Comment (nickdesaulniers) [reply boot-qemu.sh:192]: ah.  I imagine that being an issue for our CI's docker image, which is using ubuntu focal packages.

Comment (nathanchance) [reply boot-qemu.sh]: Alright, it should be fixed with 3482733422fc5be0075b802b7eb715bf78b003e2.

**Pull request merged**